pkg/sfu/pacer/base.go

```diff
@@ -83,8 +92,10 @@ func (b *Base) SendPacket(p *Packet) (int, error) {
 func (b *Base) patchRTPHeaderExtensions(p *Packet) error {
 	sendingAt := mono.Now()
 	if p.AbsSendTimeExtID != 0 {
-		absSendTime := rtp.NewAbsSendTimeExtension(sendingAt)
```

Comment:
calling this was reporting a bunch of memory in alloc_space. So, just instantiating it directly like the TWCC case below